packages/expect/src/__tests__/__snapshots__/spyMatchers.test.js.snap

```diff
@@ -177,92 +187,86 @@ Received has value: <red>[Function fn]</>"
-"<dim>expect(</><red>jest.fn()</><dim>).not.lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>not<dim>.</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to not have last returned:
-  <green>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>
-But it last returned exactly:
-  <red>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>"
+Expected value: not <green>Immutable.Map {\\"a\\": {\\"b\\": \\"c\\"}}</>
+
+Received number of returns: <red>1</>"
 `;
 
 exports[`lastReturnedWith works with Map 1`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).not.lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>not<dim>.</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
+
+Expected value: not <green>Map {1 => 2, 2 => 1}</>
 
-Expected mock function to not have last returned:
-  <green>Map {1 => 2, 2 => 1}</>
-But it last returned exactly:
-  <red>Map {1 => 2, 2 => 1}</>"
+Received number of returns: <red>1</>"
 `;
 
 exports[`lastReturnedWith works with Map 2`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to have last returned:
-  <green>Map {\\"a\\" => \\"b\\", \\"b\\" => \\"a\\"}</>
-But the last call returned:
-  <red>Map {1 => 2, 2 => 1}</>"
+Expected value: <green>Map {\\"a\\" => \\"b\\", \\"b\\" => \\"a\\"}</>
+Received value: <red>Map {1 => 2, 2 => 1}</>
+
+Received number of returns: <red>1</>"
 `;
 
 exports[`lastReturnedWith works with Set 1`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).not.lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>not<dim>.</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to not have last returned:
-  <green>Set {1, 2}</>
-But it last returned exactly:
-  <red>Set {1, 2}</>"
+Expected value: not <green>Set {1, 2}</>
+
+Received number of returns: <red>1</>"
 `;
 
 exports[`lastReturnedWith works with Set 2`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to have last returned:
-  <green>Set {3, 4}</>
-But the last call returned:
-  <red>Set {1, 2}</>"
+Expected value: <green>Set {3, 4}</>
+Received value: <red>Set {1, 2}</>
+
+Received number of returns: <red>1</>"
 `;
 
 exports[`lastReturnedWith works with argument that does match 1`] = `
-"<dim>expect(</><red>jest.fn()</><dim>).not.lastReturnedWith(</><green>expected</><dim>)</>
+"<dim>expect(</><red>jest.fn()</><dim>).</>not<dim>.</>lastReturnedWith<dim>(</><green>expected</><dim>)</>
 
-Expected mock function to not have last returned:
-  <green>\\"foo\\"</>
-But it last returned exactly:
-  <red>\\"foo\\"</>"
+Expected value: not <green>\\"foo\\"</>
```

Comment:
These two lines are confusing, it sounds a bit like the number of returns being 1 is the problem. Not sure how to improve it though

Reply:
Reading through your eyes always helps me. Unlike reports which contrast `Expected number of …` and `Received number of …` what do you think about a shorter label in these reports:

```sh
Number of returns: 1
```

Reply:
Yes, I think that'd help!